Utilities/CodeGeneration/EdFi.Ods.CodeGen/Generators/EntityOrmMappings.cs

```diff
@@ -26,6 +26,7 @@ public class EntityOrmMappings : GeneratorBase
         private readonly IDatabaseTypeTranslator _databaseTypeTranslator;
         private Dictionary<Entity, List<ClassMappingContext>> _classMappingsForEntities;
         private Func<Entity, bool> _shouldRenderEntityForSchema;
+        private readonly string _propertyAccessor = $"EdFi.Ods.Common.Infrastructure.Accessors.EmbeddedObjectPropertyAccessor, EdFi.Ods.Common";
```

Comment:
Readonly but no changes, string interpolation but no variables, is this correct?  Should we be using the Namespaces constants?